doc/sphinx/src/using-closures.rst

```diff
@@ -167,6 +298,23 @@ objects for each equation of state, similar to the vector API for a
 given EOS. Here the indexers/vectors are not over cells, but
 materials.
 
+.. warning::
+
+  It bears repeating: **both the volume fractions and densities act as inputs
+  and outputs**. They are used to define the internal :math:`\bar
+  {\rho}_i` variables at the beginning of the PTE solve. The volume fractions
+  and densities at the end of the PTE solve will represent those for the new
+  PTE state.
+
+.. warning::
+
+  The PTE solvers ***require*** that all input densities and volume fractions
```

Comment:
One extra pair of stars
```suggestion
  The PTE solvers **require** that all input densities and volume fractions
```

Reply:
I was going for italics but forgot which markdown convention Sphinx uses. However, I think that's probably not necessary. Committed your suggestion as-is